lib/msf/core/post/file.rb

```diff
@@ -53,7 +53,7 @@ def pwd
       if session.platform == 'windows'
         # XXX: %CD% only exists on XP and newer, figure something out for NT4
         # and 2k
-        return session.shell_command_token("echo %CD%")
+        return session.shell_command_token("echo %CD%").to_s.strip
```

Comment:
Curious if we need this after the changes in https://github.com/rapid7/metasploit-framework/pull/15362 landed (Specifically these three lines: https://github.com/rapid7/metasploit-framework/pull/15362/files#diff-d173cbcdf8260e163d18837d5154c57e0646ad954187bce96fae8e78105cc67fR66) where we normalize the line endings already.

Reply:
To be clear, I'm not super passionate about removing it, as it is sort of a belt/suspenders thing, and it does not hurt anything having it there.

Reply:
Hey @bwatters-r7, I just tested this and looks like we still need to strip it.
```
>> pwd
=> "C:\\Users\\User\\Links\r\n"
```